README.md

```diff
@@ -84,6 +89,7 @@ Propeller PPK uses the following EXIF headers. Other headers are permitted and w
 | `GPSLatitudeRef`    | Presenting images on a map during the upload process                       | Mandatory unless approximate position provided in the metadata.json file |
 | `GPSAltitude`       | Presenting images on a map during the upload process                       | Mandatory unless approximate position provided in the metadata.json file |
 | `GPSAltitudeRef`    | Presenting images on a map during the upload process                       | Mandatory unless approximate position provided in the metadata.json file |
+| `SerialNumber`      | Identifies the specific Camera that was used to capture images             | Mandatory unless Camera Serial Number is supplied in metadata.json file  |
```

Comment:
Can we call this `CameraSerialNumber`? Removes the ambiguity & matches DJI craft